spec/05-migration/plugins/opentelemetry/migrations/001_331_to_332_spec.lua

```diff
@@ -0,0 +1,61 @@
+
+local cjson = require "cjson"
+local uh = require "spec.upgrade_helpers"
+
+local v = os.getenv("OLD_KONG_VERSION"):sub(1,5):gsub("%.", "");
+local handler = tonumber(v) >= 330 and describe or pending
+
+
+if uh.database_type() == 'postgres' then
+    handler("opentelemetry plugin migration", function()
+        lazy_setup(function()
+            assert(uh.start_kong())
+        end)
+
+        lazy_teardown(function ()
+            assert(uh.stop_kong(nil, true))
+        end)
+
+        uh.setup(function ()
+            local admin_client = assert(uh.admin_client())
+
+            local res = assert(admin_client:send {
+                method = "POST",
+                path = "/plugins/",
+                body = {
+                    name = "opentelemetry",
+                    config = {
+                        endpoint = "http://localhost:8080/v1/traces",
+                        queue = {
+                            max_batch_size = 1
```

Comment:
I find the explicit `max_batch_size` here a bit misleading, even though it illustrates what's going on.  What we really want to test is the change of the default, so I would leave `max_batch_size` unspecified here and then verify that it was defaulted to 1.